drools-wb-services/drools-wb-verifier/drools-wb-verifier-guided-decision-table-adapter/src/main/java/org/drools/workbench/services/verifier/plugin/client/builders/ValuesResolver.java

```diff
@@ -162,28 +178,38 @@ private Values getNumericValue() throws
             ValueResolveException {
         try {
 
-            final String stringValue = realCellValue.getStringValue();
+            if (operatorFromCell.isPresent()) {
+                return resolveValue(getCellValueFromCellWithOperator());
+            } else {
+                return resolveValue(realCellValue.getStringValue());
+            }
+        } catch (final Exception e) {
+            throw new ValueResolveException("Failed to resolve String Value");
+        }
+    }
 
-            if (stringValue != null && !stringValue.isEmpty()) {
-                if (conditionColumn.getOperator() != null
-                        && (conditionColumn.getOperator()
-                        .equals("in") || conditionColumn.getOperator()
-                        .equals("not in"))) {
-                    final Values values = new Values();
+    private String getCellValueFromCellWithOperator() {
+        return realCellValue.getStringValue().trim().substring(operatorFromCell.get().length()).trim();
+    }
 
-                    for (final String item : stringValue.split(",")) {
-                        values.add(item.trim());
-                    }
+    private Values<String> resolveValue(final String stringValue) {
+        if (stringValue != null && !stringValue.isEmpty()) {
+            if (conditionColumn.getOperator() != null
+                    && (conditionColumn.getOperator()
+                    .equals("in") || conditionColumn.getOperator()
+                    .equals("not in"))) {
```

Comment:
could we please make this %% || condition a one lline method with self-explaining name to make code more readable?